src/dredd-command.coffee

```diff
@@ -266,7 +269,6 @@ class DreddCommand
       , waitMilis
 
   run: ->
-    logger.verbose('Preparing for start of testing.')
```

Comment:
I am not sure with logging policy, so just pointing this removal for reconsideration, whether it should be really removed or not.